### PR TITLE
HEAT-276 Adding team health page

### PR DESCRIPTION
**Pull request by andrewrlee**

### CODE REVIEW

File: assets/js/driftRadiator.js [hunk 1]
```diff
@@ -38,7 +38,7 @@ class DeploymentRenderer {
     components.forEach(component => {
       const environments = component.environments
         .map(environment => {
-          const gitDiffUrl = `https://github.com/ministryofjustice/${component.name}/compare/${environment.sha}...${component.devEnvSha}`
+          const gitDiffUrl = `https://github.com/ministryofjustice/${component.repo}/compare/${environment.sha}...${component.devEnvSha}`
```
Comment (andrewrlee) [inline assets/js/driftRadiator.js:41]: fix mono-repo diff link

File: assets/js/driftRadiator.js [hunk 2]
```diff
@@ -76,10 +76,6 @@ class DeploymentRenderer {
   }
 
   start = async componentNames => {
-    const update = async () => {
-      await this.fetchMessages(componentNames)
-      setTimeout(update, 10000)
-    }
-    update()
+    await this.fetchMessages(componentNames)
```
Comment (andrewrlee) [inline assets/js/driftRadiator.js:79]: removed auto refresh as it didn't play well with sorting

File: server/views/partials/docs.njk [hunk 3]
```diff
@@ -0,0 +1,44 @@
+
+{% macro drift() %}
```
Comment (andrewrlee) [inline server/views/partials/docs.njk:2]: added these doc macros to show in multiple places.. we could migrate this to dev docs and have a link once we have some dev docs set up

Comment (andrewrlee) [reply server/views/partials/docs.njk:2]: should probably run this through some LLM to tidy up the text 😅

File: server/views/partials/driftRadiatorTable.njk [hunk 4]
```diff
@@ -1,12 +1,12 @@
 
 
 {% macro driftRadiatorTable(params) %}
-    <table id="dependencyDriftTable" class="govuk-table {{ 'govuk-!-margin-top-9' if params.standalone }}">
+    <table id="dependencyDriftTable" data-module="moj-sortable-table" class="govuk-table {{ 'govuk-!-margin-top-9' if params.standalone }}">
```
Comment (andrewrlee) [inline server/views/partials/driftRadiatorTable.njk:4]: add sorting

File: assets/js/driftRadiator.js [hunk 5]
```diff
@@ -54,14 +54,14 @@ class DeploymentRenderer {
         .join('')
 
       $('#dependencyDriftRows').append(`<tr class= "govuk-table__row">
-        <td class="govuk-table__cell"><a href="/components/${component.name}" class="statusTileName">${component.name}</a></td>
-        <td class="govuk-table__cell">
+        <td class="govuk-table__cell"><a href="/components/${component.name}" class="statusTileNam govuk-link--no-visited-state">${component.name}</a></td>
```
Comment (psoleckimoj) [inline assets/js/driftRadiator.js:57]: `statusTileName` ?

Comment (andrewrlee) [reply assets/js/driftRadiator.js]: These weren't being used so removed these classes

**Review by psoleckimoj — APPROVED** [review]
LGTM!

**Pull request merged**